README.md

```diff
@@ -0,0 +1,164 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Greeting App</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
```

Comment:
- [ ] I think it would be better if you improve the content of each sub-section of the `Built With` section with appropriate and correct information. This will make your documentation be more professional and descriptive. Below is an example:
```
- Tech Stack
 - Client
    - Ruby on Rails
    - Ruby
    - React
    - Redux
 - Server
    - Localhost
 - Database
    - Postgresql
```